js/exporter/jspdf/v3/draw_utils.js

```diff
@@ -82,7 +82,14 @@ function drawCellText(doc, cell, docStyles) {
             w: _rect.w - (padding.left + padding.right),
             h: _rect.h - (padding.top + padding.bottom)
         };
+        if(isDefined(cell._textOffset)) {
+            textRect.x = textRect.x + cell._textOffset.left;
```

Comment:
'cell._textOffset.left;' - do we really need a complex object to provide a single Number value ?

Reply:
Ok, i make the simple number value.
In the Future if we neccessary expand this value to complex. We will do it.